src/react/components/Button.js

```diff
@@ -13,7 +13,6 @@ const Button = ({ style = {}, data, active, onClick }) => (
   <div
     role="button"
     onClick={onClick}
-    onKeyDown={onClick}
```

Comment:
What was purpose of this onKeyDown in the first place? Do we lose some feature now as it is removed?

Reply:
eslint proposed it, if I remember right.

Reply:
And that fixed the "Unintentional key press on focus" issue?
One line fix? 😳 👌